gpt4all-chat/embllm.cpp

```diff
@@ -303,6 +303,18 @@ EmbeddingLLM::~EmbeddingLLM()
     m_embeddingWorker = nullptr;
 }
 
+QString EmbeddingLLM::model() const
+{
+    const EmbeddingModels *embeddingModels = ModelList::globalInstance()->installedEmbeddingModels();
+    if (!embeddingModels->count())
+        return QString();
+
+    const ModelInfo defaultModel = embeddingModels->defaultModelInfo();
+    if (!defaultModel.name().isEmpty())
+        return defaultModel.name();
+    return QString();
```

Comment:
This can be simplified:
```suggestion
    return defaultModel.name(); // empty name is an error
```